octoprint_mrbeam/__init__.py

```diff
@@ -139,7 +139,7 @@ class MrBeamPlugin(
     ENV_LASER_SAFETY = "laser_safety"
     ENV_ANALYTICS = "analytics"
 
-    LASERSAFETY_CONFIRMATION_DIALOG_VERSION = "0.4"
+    LASERSAFETY_CONFIRMATION_DIALOG_VERSION = "0.5"
```

Comment:
Will this show the dialoge angain for all that already confirmed it?

Reply:
This basically is just for documentation and nothing else

Reply:
It is used in the payload uploaded to the cloud but it won't trigger another confirmation